Download.Mvc/Client/components/Search.tsx

```diff
@@ -49,7 +56,10 @@ export class Search extends React.Component<IProps, any> {
         return (
             <div>
                 <SearchBar />
-                <h3>Searching for: {decodeURIComponent(this.props.match.params.query)}</h3>
+                {this.props.match.params.query &&
+                    <h3>Searching for: {decodeURIComponent(this.props.match.params.query)}</h3>}
+                {!this.props.match.params.query &&
+                    <h3>Search for: </h3>}
```

Comment:
@laholstege why do you need to do this, won't decoding an empty string just give you the same empty string back?  Also do you mean to change the language up Searching/Search for?